packages/oruga-next/src/components/autocomplete/Autocomplete.vue

```diff
@@ -442,7 +442,7 @@ const isEmpty = computed(
 );
 
 watch(isEmpty, (empty) => {
-    if (isFocused) isActive.value = !empty;
+    if (isFocused.value) isActive.value = !empty || !!slots.empty;
```

Comment:
Nice bug you found here :D Could you exclude it in its own PR? it makes the changelog clearer.

Reply:
Yes, this can be broken out cleanly.

Reply:
Now tracked as https://github.com/oruga-ui/oruga/pull/787